documentation/howtos.md

```diff
@@ -4,58 +4,125 @@ By default, instances are not provisioned with either ecommerce or the Course Di
+  hour: "1"
+  minute: "43"
+  day: "*"
+```
+
 * Set these extra variables on the instance:
 
-        SANDBOX_ENABLE_DISCOVERY: yes
-        SANDBOX_ENABLE_ECOMMERCE: yes
-        DISCOVERY_VERSION: 'open-release/ginkgo.1'
-        nginx_discovery_gunicorn_hosts:
-            - "127.0.0.1"
-        COMMON_HOSTNAME: "your-site-name"
-        ECOMMERCE_PAYMENT_PROCESSOR_CONFIG:
-            your-partner-code:
-              paypal:
-                ...
+```
+# prevent creating example partners; this isn't required, and will cause provision errors later
+ecommerce_create_demo_data: false
+
+SANDBOX_ENABLE_DISCOVERY: yes
+SANDBOX_ENABLE_ECOMMERCE: yes
+DISCOVERY_VERSION: 'open-release/ginkgo.1' # match version to edx version
+nginx_discovery_gunicorn_hosts:
+    - "127.0.0.1"
+COMMON_HOSTNAME: ""
+ECOMMERCE_PAYMENT_PROCESSOR_CONFIG:
+    your-partner-code: # this is arbitrary; will be used later
+      paypal:
+        ...
+```
 
 **Notes**:
 
-* We need to set the `COMMON_HOSTNAME` to something other than the FQDN, so
+* We need to set the `COMMON_HOSTNAME` to something other than the external
+  lms domain name (eg. `openlearning.example.com`), so
   that API requests made on the server can be properly routed through the load
-  balancer-terminated SSL connection.  This is required because, by default, the
-  appserver's FQDN refers to localhost, which doesn't understand SSL.
+  balancer-terminated SSL connection.  This is required because, by default, an
+  /etc/hosts entry for the `COMMON_HOSTNAME` is set pointing to localhost,
+  which will break SSL connections to the LMS from within the instance.
 * The [`ECOMMERCE_PAYMENT_PROCESSOR_CONFIG`](https://github.com/edx/configuration/blob/d68bf51d7b8403bdad09dc764af5ebafe16d7309/playbooks/roles/ecommerce/defaults/main.yml#L103)
   should contain the payment processors and their keys.
 
+
+* A larger appserver vm type may need to be used, since it will be running
+  ecommerce and course discovery as well.
+
 Once the spawn is complete, you'll need to take the following steps to finish setup
 (these are one-time actions that are stored in the database):
 
 1. Create/choose a staff user to use for the OAuth2 clients.
    Ensure the staff user has a user profile associated (i.e. set a Full Name).
-1. In Django Admin > OAuth2 > Clients, there should already be clients created
-   for ecommerce and discovery.
+1. In Django Admin > OAuth2 > Clients (`/admin/oauth2/client/`), there should
+   already be clients created for ecommerce and discovery.
    If not, [create and register new clients](http://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/ecommerce/install_ecommerce.html#configure-edx-openid-connect-oidc)
    for each service.  You'll need the client IDs and client secrets for the next
-   steps.
-   Ensure that both clients are attached to the staff user updated above.
+   steps.  Ensure that both clients are attached to the staff user updated above.
```

Comment:
The `ecommerce_worker` user isn't a staff user, which might be related to the permissions issue I'm having with creating course seats in ecommerce?  I tried elevating the user and it didn't fix it though.  Also, I think reprovisioning might re-set this back to a non-staff user.

Reply:
The issue was that some permissions were missing for ecommerce_worker. Elevating to staff doesn't override permissions granted; I think only superuser does that.

Reply:
Huh interesting.. Thanks for [adding those](https://github.com/open-craft/opencraft/pull/523/files#diff-e62c375eb49e96b787ac278c5d78e70dR76)!